packages/main/src/plugin/file-tree.spec.ts

```diff
@@ -0,0 +1,66 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
```

Comment:
```suggestion
 * Copyright (C) 2024 Red Hat, Inc.
```

it will start in 2024 when it's merged